src/components/Card.js

```diff
@@ -43,18 +64,18 @@ class Card {
       .classList.toggle("card__like-button_active");
   }
 
-  // _handlePreviewPicture() {
-  //   cardPreviewImage.src = this._link;
-  //   cardPreviewImage.alt = this._name;
-  //   cardPreviewTitle.textContent = this._name;
-  //   openModal(cardImageModal);
-  // }
+  deleteCard() {
+    this._handleDeleteCard();
+  }
 
   getView() {
     this._cardElement = document
       .querySelector(this._cardSelector)
       .content.querySelector(".card")
       .cloneNode(true);
+
+    this._likeButton = this._cardElement.querySelector(".card__like-button");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**

Reply:
thank you